apps/approval/models.py

```diff
@@ -30,6 +32,22 @@ class Approval(models.Model):
     approved = models.BooleanField(_("godkjent"), default=False, editable=False)
     message = models.TextField(_("melding"))
 
+    def alert_user():
+        if approved:
+            header_message = "Ditt medlemskap i Online er godkjent"
+        else:
+            header_message = "Ditt medlemskap i Online er ikke godkjent"
+
+        email = applicant.get_email()
+        if (not email == None):
+            send_mail(
+                header_message,
+                header_message,
```

Comment:
The message could be different from the header :-)

It should also include the comment regarding why the membership was denied.